packages/mobile_app/lib/components/prophecy_record.dart

```diff
@@ -1,12 +1,11 @@
 import 'package:flutter/material.dart';
 import 'package:prophecy_model/prophecy_model.dart';
+import 'package:app/theme/app_colors.dart';
 
-import '../theme/app_colors.dart';
-
-class ProphecyRecordWidget extends StatelessWidget {
+class ProphecyRecord extends StatelessWidget {
```

Comment:
Just out of curiosity, why removing prefix *Widget*?

Reply:
because it is in /components folder.